.github/workflows/editorconfig.yml

```diff
@@ -1,19 +1,20 @@
-name: actions
+name: "Checking EditorConfig"
 
 on:
   pull_request:
     branches:
       - master
 
 jobs:
-  editorconfig:
+  tests:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: technote-space/get-diff-action@v1.2.8
-    - name: editorconfig check
+    - uses: technote-space/get-diff-action@v2.0.2
+    - name: Checking EditorConfig
+      if: github.repository_owner == 'NixOS'
```

Comment:
does it not make sense to check this in forks too?

Reply:
Wouldn't we only want to run it here and not in forks?

Reply:
The advantage would be that they could fix it even before the pull request is opened.

Reply:
Maybe I'm misunderstanding this specific condition? I've seen similar checks to prevent actions running when not in the original repo?

Reply:
Reverted.

Reply:
Yes it prevents this action to be run on forks. This is actually my intention. People should be able to test their code before hand.